### PR TITLE
changing match uuid logic try duration from 1 to 3 secs

### PR DESCRIPTION
**Pull request by 2infinitee**
increasing duration of sleep for each try from 1 to 3 secs before loop errors out.

### CODE REVIEW

File: pkg/checks/external/main.go [hunk 1]
```diff
@@ -354,8 +354,8 @@ func (ext *Checker) setUUID(uuid string) error {
 			return fmt.Errorf("failed to verify uuid match %s after %d tries", checkState.Spec.CurrentUUID, tries)
 		}
 		tries++
-		ext.log("Waiting 1 second before checking " + ext.Name() + " uuid.")
-		time.Sleep(time.Second)
+		ext.log("Waiting 3 second before checking " + ext.Name() + " uuid.")
+		time.Sleep(time.Second * 3)
```
Comment (jonnydawg) [inline pkg/checks/external/main.go:358]: I think we should up the limit of retries instead of upping the time in-between retries.

Comment (integrii) [reply pkg/checks/external/main.go:358]: Agree with @jonnydawg .  Could you change this on the `tries` variable instead please?  That way we don't add so much extra delay here...

Maybe set `tries` to a maximum of 9 and drop the sleep back down to 1s.

**Comment by 2infinitee** [comment]
@integrii merged master into this branch and it seemed to resolved all checks. Feel free to merge.

File: pkg/checks/external/main.go (hunk 1 above)
Comment (2infinitee) [reply pkg/checks/external/main.go]: done, changed from 3 secs to 1 and also up the number of tries to 9

**Pull request merged**